cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1069,9 +1073,7 @@ function Configure_Dockerd_CniNetworking {
   replace('DNS_SERVER_IP', ${kube_env}['DNS_SERVER_IP']).`
   replace('DNS_DOMAIN', ${kube_env}['DNS_DOMAIN']).`
   replace('MGMT_IP', ${mgmt_ip}).`
-  replace('CLUSTER_CIDR', ${kube_env}['CLUSTER_IP_RANGE']).`
```

Comment:
Remove CLUSTER_IP_RANGE in the comment at line 987. Same for the containerd function.

Reply:
removed